kubernetes/pre_post_install_task_generator.py

```diff
@@ -0,0 +1,38 @@
+import re
```

Comment:
Add `python` requirement section the documentation, any specific version?

Reply:
Done